examples/step-57/doc/intro.dox

```diff
@@ -0,0 +1,319 @@
+Foundation grant DMS1522191 and the Computational Infrastructure in
+Geodynamics initiative (CIG), through the National Science Foundation under
+Award No. EAR-0949446 and The University of California-Davis.
+</i>
+
+<a name="Intro"></a> 
+<h1>Introduction</h1>
+
+<h3> Navier Stokes Equations </h3>
+
+In this tutorial we show how to solve the incompressible Navier
+Stokes equations (NSE) by Newton's method. The flow we consider here
+is assumed to be steady. In a domain $\Omega \subset
+\mathbb{R}^{d}$, $d=2,3$, with a piecewise smooth boundary
+$\partial \Omega$, and a given force field $\textbf{f}$, we seek
+a velocity field $\textbf{u}$ and a pressure field $\textbf{p}$
+satisfying
+
+@f{eqnarray*}
+- \nu \Delta\textbf{u} + (\textbf{u} \cdot \nabla)\textbf{u} + \nabla p &=& \textbf{f}\\
+- \nabla \cdot \textbf{u} &=& 0
+@f}
+
+Different from the Stokes equations as discussed in step-22, the NSE are a
+nonlinear system because of the convective term $(\textbf{u} \cdot
+\nabla)\textbf{u}$. The first step of computing a numerical solution
+is to linearize the system and this will be done using Newton's method. A
+time-dependent problem is discussed in step-35, where the system is linearized
+using the solution from the last time step and no nonlinear 
+solve is necessary.
+
+<h3> Linearization of Navier Stokes Equations </h3>
+
+Moving the right-hand side terms to the left, we rewrite the NSE as
+
+@f{eqnarray*}
+F(\mathbf{u}, p) = \left(
+                \begin{array}{c}
+                  - \nu \Delta\mathbf{u} + (\mathbf{u} \cdot \nabla)\mathbf{u} + \nabla p - \mathbf{f} \\
+                  - \nabla \cdot \mathbf{u} \\
+                \end{array}
+              \right).
+@f}
+
+$F(\textbf{u}, p)$ is a nonlinear function whose root is 
+the solution to the NSE. Assuming the initial guess is good enough to
+guarantee the convergence of Newton's iteration and denoting
+$\textbf{x} = (\textbf{u}, p)$, Newton's iteration on a vector field
+can be defined as
+@f{eqnarray*} \textbf{x}^{k+1} = \textbf{x}^{k} -
+(\nabla F(\textbf{x}^{k}))^{-1} F(\textbf{x}^{k}),
+@f}
+where
+$\textbf{x}^{k+1}$ is the approximate solution in step k+1,
+$\textbf{x}^{k}$ represents the solution from the last step, and $\nabla
+F(\textbf{x}^{k})$ is the Jacobian matrix evaluated at
+$\textbf{x}^{k}$.
+
+From Newton's iteration formula, we can observe that the new
+solution is obtained by adding an update term to the old solution. Instead
+of evaluating the Jacobian matrix and taking its inverse, we consider
+the update term as a whole, that is
+
+@f{eqnarray*}
+\delta \textbf{x}^{k} = - (\nabla F(\textbf{x}^{k}))^{-1} F(\textbf{x}^{k}),
+@f}
+where $x^{k+1}=x^{k}+\delta x^{k}$.
+
+Next, we solve the system
+@f{eqnarray*}
+\nabla F(\textbf{x}^{k}) \delta \textbf{x}^{k} = -F(\textbf{x}^{k})
+@f}
+for the update term $\delta \textbf{x}^{k}$. Here, $\nabla F(\textbf{x}^{k})$ represents the
+directional gradient of $F(\textbf{x})$ along $\delta
```

Comment:
Well, that's not quite true. You need to have the delta x in the first formula of the sentence then.